src/components/todo-input/TodoInput.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <form class="input-model">
+    <v-text-field
+      class="text-placeholder"
+      placeholder="what needs to be done?"
+      clearable
+      v-model="textValue"
+    >
+      <v-btn icon slot="prepend-inner">
+        <v-icon>
+          {{ icons.mdiChevronDown }}
+        </v-icon>
+      </v-btn>
+
+      <v-icon @click="onAddTodo" v-if="!!textValue" icon slot="append">
```

Comment:
When there are more than two props please use multiline
```
<v-icon 
  v-if="!!textValue"
  icon
  slot="append"
  @click="onAddTodo"
>
```

Reply:
The rule should be like this, directives first (v-model, v-if), properties (icon, slot), event handlers (@click, @keyup)